.travis.yml

```diff
@@ -3,10 +3,10 @@ notifications:
   recipients:
     - eduardo@gurgel.me
 elixir:
-  - 1.4.2
+  - 1.5.3
 otp_release:
   - 18.0
-  - 18.1
   - 19.0
+  - 20.0
```

Comment:
Maybe use a [build matrix](https://docs.travis-ci.com/user/build-matrix/) to test on newer OTP versions with supported elixir versions as well? like in https://github.com/edgurgel/httpoison/blob/master/.travis.yml